data.sql

```diff
@@ -212,4 +212,7 @@ VALUES
   WHERE v.name = 'William Tatcher'
     AND a.name = 'Blossom';
 
-  
\ No newline at end of file
+  
+  CREATE INDEX idx_visits_animal_id ON visits (animal_id);
+  CREATE INDEX idx_visits_vet_id ON visits (vet_id);
+  CREATE INDEX idx_owners_email ON owners (email);
```

Comment:
- [ ] This code on lines 216 to 218 should be done in the schema file since it alters the database structure. Please fix it